cli/dts/lib.deno.ns.d.ts

```diff
@@ -113,26 +113,48 @@ declare namespace Deno {
    * See: https://no-color.org/ */
   export const noColor: boolean;
 
-  export interface TestDefinition {
-    fn: () => void | Promise<void>;
+  export class Tester {
```

Comment:
I was just implementing the RFC as-is, but I don't think we should bother having this as an exposed class to the user. Maybe just make it an interface and don't export it?

Reply:
Unsound to have as an exported class, test ops are only available with test command.

Reply:
Yeah - let's not export it.